Makefile

```
@@ -448,21 +448,46 @@ ${TOOLCHAIN_DIR}/snitch_cluster:
 	cd ${TOOLCHAIN_DIR} && \
 	git clone https://github.com/pulp-platform/snitch_cluster.git && \
 	cd ${TOOLCHAIN_DIR}/snitch_cluster && git checkout ${SNITCH_COMMIT_HASH} && \
-	git submodule update --init --recursive && \
-	git checkout ${SNITCH_COMMIT_HASH} && git apply ${TOOLCHAIN_DIR}/snitch_cluster.patch
+	git -c url."https://github.com/".insteadOf="git@github.com:" \
+	    submodule update --init --recursive && \
+	git apply --3way ${TOOLCHAIN_DIR}/snitch_cluster.patch
 
 ${SNITCH_INSTALL_DIR}: ${TOOLCHAIN_DIR}/snitch_cluster
```

checkmake (0.3.2)

[warning] Target body for "${SNITCH_INSTALL_DIR}" exceeds allowed length of 5 lines (9).

(maxbodylength)